justfile

```diff
@@ -44,7 +44,7 @@ test-count:
 lint:
   env NO_STASH=true misc/git-hooks/pre-commit
   just clippy
-  env RUSTDOCFLAGS='-D rustdoc::broken_intra_doc_links' cargo doc --profile dev --no-deps --document-private-items
+  env RUSTDOCFLAGS='-D rustdoc::broken_intra_doc_links -D warnings' cargo doc --profile dev --no-deps --document-private-items
```

Comment:
is -D rustdoc::broken_intra_doc_links still needed?

Reply:
Not sure, but let's just keep it.